README.md

```diff
@@ -94,6 +94,23 @@ export SMTP_USERNAME=<your-username>
 export SMTP_PASSWORD=<your-password>
 ```
 
+##### Trigger a breach alert email
+To trigger a breach alert email, you need to simulate a POST to the
+`/hibp/notify` endpoint. The endpoint requires:
+* POST request of `application/json` type
```

Comment:
Add a line break here so that it renders correctly into a list.

Reply:
Done. Also cleaned up language to be a bit more consistent.